libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -52,10 +52,16 @@ + (NSString *)localizedString:(NSString *)localizationKey
 
 + (UIImage *)imageNamed:(NSString *)name
 {
-    NSBundle *bundle = [NSBundle bundleForClass:[self class]];
     NSAssert(name != nil, @"name must contain a value.");
+    //Get image from main bundle (this is to allow extracting SDK Assets when using template app as we include SDK Assets as part of the tempalte app)
```

Comment:
The solution is inaccurate.  There are stumbling blocks to getting asset catalogs even properly recognized in CocoaPods, and then [bugs/limitations in CocoaPods itself](https://github.com/CocoaPods/CocoaPods/issues/1546) that do not allow them to be compiled properly, once they exist in the Pod.

I think the proper support for the time being, given our heavy reliance on CocoaPods coupled with its lack of proper support for asset catalogs, is to deliver our image assets directly, through a resource bundle.  I.e. the old-fashioned way.

There's some elusive logic in between the lines out there for how you can make asset catalogs work with CocoaPods, but in my opinion, the moment you have to piece together 4 separate bug reports, 3 blog posts, and a Twitter post, to try to make a sane solution, it's time to fall back. :)